client/console.py

```diff
@@ -2,7 +2,6 @@
 
 import json
 
-from main import GameInterface
```

Comment:
Instead of removing types like this, you can put this behind a `if typing.TYPE_CHECKING` comment